### DISCUSSION TITLE
Nuxt3 dynamic nested routes shown pages not found!

### DISCUSSION THREAD

**Discussion by sikeat7** [nuxt/nuxt#17634] [General]
Hi! I'm create pages with nested route but it's not working. Example of url: http://localhost/app/dynamic_id/show/dymanic_id2
`Dynamic params {
	dynamic_id,
	dynamic_id2
}`

Folder Structure:
<img width="335" alt="Screen Shot 2022-05-05 at 4 48 50 PM" src="https://user-images.githubusercontent.com/12691222/166899858-55170c4f-beca-4d8a-b5b5-ef7fab0244a1.png">

**Results**:
**Page not found: /app/100/show/3436**

Does anyone know how to do it ? Thank you.

**Comment by videmann** [comment]
hi! we are using '[...catalog].vue' inside 'catalog' folder.
for example, our dynamic route: '/catalog/aksessuary/perfumery/155255/'
